packages/enhanced_gradients/README.md

```diff
@@ -0,0 +1,46 @@
+# enhanced_gradients
+
+This package adds an easy way to enhance your Flutter gradients by interpolating colors
```

Comment:
It's not only for gradients but also for any color transitions, most commonly components' state changes

Reply:
Well you're right! I've completely forgotten about this use case (and the HctColorTween from which the entire project started)